src/main/java/de/uhd/ifi/se/decision/management/jira/view/decisiontable/DecisionTable.java

```diff
@@ -89,16 +89,19 @@ public void setIssues(FilterSettings filterSettings, ApplicationUser user) {
 	 *            authenticated Jira {@link ApplicationUser}.
 	 */
 	public void setDecisionTableForIssue(KnowledgeElement rootElement, ApplicationUser user) {
-		Set<Link> outgoingLinks = this.graph.outgoingEdgesOf(rootElement);
 		decisionTableData.put("alternatives", new ArrayList<DecisionTableElement>());
 		decisionTableData.put("criteria", new ArrayList<DecisionTableElement>());
 
-		for (Link currentLink : outgoingLinks) {
-			KnowledgeElement targetElement = currentLink.getTarget();
-			if (targetElement.getType() == KnowledgeType.ALTERNATIVE
-					|| targetElement.getType() == KnowledgeType.DECISION) {
-				decisionTableData.get("alternatives").add(new Alternative(targetElement));
-				getArguments(targetElement);
+		// TODO Check link type. A decision that leads to a new decision problem should
+		// not be shown as solution option for this derived decision problem.
+		for (Link currentLink : graph.edgesOf(rootElement)) {
+			KnowledgeElement oppositeElement = currentLink.getOppositeElement(rootElement);
+			System.out.println(oppositeElement);

```

Comment:
Should there be printed lines in the Java Code?

Reply:
No, there should be no printed lines in the Java Code, absolutely not!